lib/csv/row.rb

```diff
@@ -172,8 +172,8 @@ def []=(*args)
     #   <<( header_and_field_hash )
     #
     # If a two-element Array is provided, it is assumed to be a header and field
-    # and the pair is appended.  A Hash works the same way with the key being
-    # the header and the value being the field.  Anything else is assumed to be
+    # and the pair is appended. A Hash works the same way with the key being
+    # the header and the value being the field. Anything else is assumed to be
     # a lone field which is appended with a +nil+ header.
```

Comment:
```Anything else is assumed to be a lone field which is appended with a +nil+ header.```

Is `lone field` correct? I was wondering if it should be `line field`.

Reply:
`lone field` is correct: "being alone" in https://www.dictionary.com/browse/lone

But we can use other wording.

Reply:
gotcha! Thanks for the link @kou 👍